app/views/catalog/_show_sections/_default_contents.html.erb

```diff
@@ -43,7 +35,7 @@
                         <% child.xpath('file').each do |file| %>
                           <li class="file">
                             <span class="label">File</span>
-                            <% if can_manage %>
+                            <% if can? :manage_content, object %>
```

Comment:
There was slack discussion around defaults, but perhaps it doesn't apply here.  The partial name indicates it is about 'content' so that's good enough for me.